python/tank/commands/setup_project_wizard.py

```diff
@@ -396,8 +407,11 @@ def get_default_configuration_location(self):
 
         if not data:
             # we are not based on an existing project. Instead pick the last primary config
+            # that is using storages (e.g. has got tank_name set)
             data = self._sg.find_one("PipelineConfiguration",
-                                     [["code", "is", "primary"]],
+                                     [["code", "is", "primary"],
```

Comment:
Dumb question but... is Shotgun's code field case insensitive?

Reply:
yes, i believe so